core/src/pipeline.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright 2018 Robin Linden <dev@robinlinden.eu>
```

Comment:
We should use `Copyright 2018 Evil Corp Ltd. contributors` or something like it.

Reply:
How about I fix all headers in another pull request?

Reply:
Fair enough. We're not changing the old ones though, just adding a new line as we touch the files while representing this org, so the files added will only be (c) Evil Corp Contributors and the old files will have two lines. That's why I only mentioned it on the added file. :P